airbyte-cdk/python/bin/low-code-unit-tests.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-set -e
+# Ideally we'd like to have set -e, but when set -e is used, the `python -m pytest unit_tests` command can return a
+# non-zero exit code and end the whole script prematurely
+# set -e
```

Comment:
Should we just remove those lines entirely if it's just commented and we don't intend to change this? I think the nature of this script makes it not needed

Reply:
Normally I'm all about getting rid of commented-out code but in this case, IMO a comment explaining why we don't want to use `set -e` is kind of nice to have because my tendency would be to add it if I was updating a script where it felt like it should be used.